backend/Classes/Room.js

```diff
@@ -421,13 +437,8 @@ method.updatePlaylists = async function() {
 * @param {string} playlistId The id that identifies the playlist
 * @return {boolean} True if completed
 */
-method.getRandomTracks = async function(playlistId, activeTrack) {
-    let playlist = this.getPlaylistById(playlistId);
-    //Load tracks into Playlist if its empty
-    if (!Array.isArray(playlist.tracks)) {
-        playlist.tracks = await this.user.fetchPlaylistTracks(playlist);
-    }
-
+method.getRandomTracks = function(playlist, activeTrack) {
```

Comment:
why isnt this async anymore?

Reply:
I changed this function so it does not have to make any api calls anymore, the earlier version would (in some cases) make a call to get the tracks of a playlist, i moved the api call somewhere else